### PR TITLE
chore/ch4556/document new software development lifecycle

### PR DESCRIPTION
**Pull request by theantichris**
- refactor: removed backlog, refined other steps
- feature: added notes about libraries
- feature: added backlog and updated qa process

### CODE REVIEW

**Review by racheljay — DISMISSED** [review]

**Review by ryexley — APPROVED** [review]
Thanks for updating this 🙇🏼

**Review by ryexley — APPROVED** [review]

**Review by racheljay — APPROVED** [review]

File: docs/software-development-lifecycle.mdx [hunk 1]
```diff
@@ -3,40 +3,51 @@ id: software-development-lifecycle
...
+    * When a story has been deemed ready it will be moved to **Ready For Development**
 1. Ready For Development
-    * These are stories that are ready to be worked on
-    * All stories in this state should be pointed
-    * Stories in this state will be scheduled to be worked on during a current or future interation
-    * When a developer is ready for a new story they will take one from this list that is assigned to them and for the current interation
-    * If there are no stories assigned to the developer they can pick an unassigned story
+    * These are stories that are ready to be worked on by engineering
+    * All stories in this state should have story points
+    * When an engineer is ready for a new story they will take one from this column this column and assign it to the current iteration
```
Comment (jmosley5) [inline docs/software-development-lifecycle.mdx:17]: "this column" appears twice

**Review by jmosley5 — CHANGES_REQUESTED** [review]
Looks good other than the one change requested 👍🏼

File: docs/software-development-lifecycle.mdx [hunk 2]
```diff
@@ -3,40 +3,51 @@ id: software-development-lifecycle
...
+    * Creating a Git branch referrencing the story ID will automatically move the story to the **In Development** column
```
Comment (rlmead) [inline docs/software-development-lifecycle.mdx:18]: Typo: "referrencing" -> "referencing"

File: docs/software-development-lifecycle.mdx [hunk 3]
```diff
@@ -3,40 +3,51 @@ id: software-development-lifecycle
...
-    * The developer opens a Pull Request from the feature branch into develop
-    * When testing, QA moves the card to the QA Testing column and will manually test the story and create the test cases that will define the Automation work after the card is moved to Accepted
-    * If QA finds issues they will add them to the story and move the card to the Rejected column
-    * Once there are no issues related to the story, QA moves the card to the QA Done column
-    * Once the Pull Request has approval from QA and the other requested developers it is merged into `develop` and moved to the Acceptance column
+    * These are stories that are actively being worked on by an engineer
+    * An engineer should normally have no more than 1 story in this column at a time
+1. QA Ready
+    * Once an engineer has finished work on a story it is moved to the QA Ready column
+    * The enginer opens a Pull Request from their feature branch into `develop`
```
Comment (rlmead) [inline docs/software-development-lifecycle.mdx:24]: Typo: "enginer" -> "engineer"

**Review by racheljay — DISMISSED** [review]

**Review by jmosley5 — APPROVED** [review]
:shipit:

**Review by racheljay — APPROVED** [review]

**Review by ryexley — APPROVED** [review]

**Pull request merged**